lib/ui/platform_dispatcher.dart

```diff
@@ -168,6 +168,23 @@ class PlatformDispatcher {
   // A map of opaque platform view identifiers to view configurations.
   final Map<Object, ViewConfiguration> _viewConfigurations = <Object, ViewConfiguration>{};
 
+  /// The implicit [FlutterView] provided by the platform, if any.
+  ///
+  /// This is used to bootstrap the framework for applications where the
+  /// platform provides a view, such as applications designed for
+  /// single-display mobile devices.
+  ///
+  /// While the [FlutterView]'s properties may change, this reference is
+  /// guaranteed to never change.
```

Comment:
nit: maybe add: "... for the lifetime of the application."

Reply:
Added